core/tests/protractor/collections.js

```diff
@@ -33,8 +33,6 @@ describe('Collections', function() {
     adminPage.get();
     adminPage.reloadCollection();
     general.acceptAlert();
-    browser.waitForAngular();
```

Comment:
This was just removed because it wasn't necessary? (Or was there some other reason?)

Reply:
Wasn't necessary; reloading a collection automatically reloads the explorations it refers to (otherwise there would be a load error).

Reply:
Apparently this is necessary for the collection editor test. :) I didn't read through it carefully enough, and since I was testing the player test in isolation I didn't realize removing this lines broke the other test.